boards/opentitan/earlgrey-cw310/layout.ld

```diff
@@ -14,36 +14,46 @@ MEMORY
    * See https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/lib/base/static_critical.ld
    * for details
    */
-  ram   (!rx) : ORIGIN = 0x10000650, LENGTH = 0x10000 - 0x650
+  ram   (!rx) : ORIGIN = 0x10000650, LENGTH = 0x20000 - 0x650
 }
 
 SECTIONS {
     .manifest ORIGIN(rom):
     {
-      _manifest = .;
-      /* sw/device/silicon_creator/lib/manifest.h */
-      . += 384; /* signature */
-      . += 4;   /* usage_constraints.selector_bits */
-      . += 32;  /* usage_constraints.device_id */
-      . += 4;   /* usage_constraints.manuf_state_creator */
-      . += 4;   /* usage_constraints.manuf_state_owner */
-      . += 4;   /* usage_constraints.life_cycle_state */
-      . += 384; /* modulus */
-      . += 4;   /* address_translation */
-      . += 4;   /* identifier */
-      . += 4;   /* length */
-      . += 4;   /* version_major */
-      . += 4;   /* version_minor */
-      . += 4;   /* security_version */
-      . += 8;   /* timestamp */
-      . += 32;  /* binding_value */
-      . += 4;   /* max_key_version */
-      . += 4;   /* code_start */
-      . += 4;   /* code_end */
-      LONG(_stext - ORIGIN(rom)); /* . = . + 4; entry_point */
-      . += 128; /* padding */
-      /* size = 1024 bytes */
+        _manifest = .;
+        /* see: sw/device/silicon_creator/lib/manifest.h */
+        . += 384; /* rsa_signature */
+        . += 4;   /* usage_constraints.selector_bits */
+        . += 32;  /* usage_constraints.device_id */
+        . += 4;   /* usage_constraints.manuf_state_creator */
+        . += 4;   /* usage_constraints.manuf_state_owner */
+        . += 4;   /* usage_constraints.life_cycle_state */
+        . += 384; /* rsa_modulus */
+        . += 4;   /* address_translation */
+        . += 4;   /* identifier */
+        . += 4;   /* manifest_version */
+        . += 4;   /* signed_region_end */
+        . += 4;   /* length */
+        . += 4;   /* version_major */
+        . += 4;   /* version_minor */
+        . += 4;   /* security_version */
+        . += 8;   /* timestamp */
+        . += 32;  /* binding_value */
+        . += 4;   /* max_key_version */
+        . += 4;   /* code_start */
+        . += 4;   /* code_end */
+        LONG(_stext - ORIGIN(rom)); /* . = . + 4; entry_point */
+        /* manifest extension table */
+        /* see: sw/device/silicon_creator/lib/base/chip.h */
+        /* CHIP_MANIFEST_EXT_TABLE_COUNT = 15 */
+        /* sizeof(manifest_ext_table_entry) = 8 */
+        . += 120; /* manifest_ext_table */
     } > rom
+
+    .vectors : ALIGN (256) {
+      KEEP(*(.riscv.open_titan_trap_vectored));
+    } > rom
```

Comment:
What's this doing? There's no section named `.riscv.open_titan_trap_vectored`. There is `.riscv.trap_vectored` but it's alignment is already handled in kernel_layout.ld.

Reply:
Good catch.  Removed.